runbotics/runbotics-orchestrator-ui/src/translations/en/components.json

```diff
@@ -17,6 +17,7 @@
     "Component.Tile.Process.Content.Creator": "Creator:",
     "Component.Tile.Process.Content.Updated": "Updated:",
     "Component.Tile.Process.Content.Scheduled": "Scheduled:",
+    "Component.Tile.Process.Content.Description": "Description:",
```

Comment:
if it is what I think (label on process tile) then we should not include it, just change font size to greyish and boom we have text of lower importance and it is obvious it is description

Reply:
description is only visible for external user